module/os/freebsd/spl/list.c

```diff
@@ -27,10 +27,12 @@
  * Generic doubly-linked list implementation
  */
 
+#include <sys/ccompile.h>
```

Comment:
Yea... this run against the very strong preference in the rest of stand to rarely or never use -include because of the problems we've had with it in the past...  However, this one file likely can be accommodated...  I'll tidy up these includes..